pyne/bins.pyx

```diff
@@ -118,19 +118,66 @@ def pointwise_linear_collapse(np.ndarray[np.float64_t, ndim=1] x_g,
     y_g : np.ndarray
         The group collapsed data, length G. 
     """
+    return pointwise_collapse(x_g, x, y, (False, False))
+
+@cython.boundscheck(False)
+def pointwise_collapse(np.ndarray[np.float64_t, ndim=1] x_g, 
+                       np.ndarray[np.float64_t, ndim=1] x, 
+                       np.ndarray[np.float64_t, ndim=1] y,
+                       log=(False, False)):
+    """Collapses pointwise data to G groups based on a interpolation
+    between the points. This is useful for collapsing cross section data.
+
+    Parameters
+    ----------
+    x_g : array-like
+        Group boundaries, length G+1 for G groups, must be monotonic in the 
+        same direction as x.
+    x : array-like
+        Pointwise abscissa to be collapsed, must be monotonic in the same direction
+        as x_g and have the same length as y.
+    y : array-like
+        Pointwise data to be interpolated, must have the same length as x.
+    log : tuple of bools, optional, default = (False, False)
```

Comment:
Should ```log```, ```logx```, ```logy``` be bools? If they aren't and they are set to ```None```, then what are they set to to turn them on?